### PR TITLE
Eu opinions srg history field

### PR DESCRIPTION
**Pull request by Levia**
## Description

Initial implementation related to adding the new SRG History field.
It currently adds:

* New srg_histories database table with `name` and `tooltip`
* Ability to add/edit/destroy SRG histories from the admin interface
* Ability to select an SRG history from a dropdown in the EU Opinions form
* Add SRG Referral decision type

## Notes
Migrate needed

### CODE REVIEW

**Comment by Levia** [comment]
Merged here the most recent changes related to removing the presence validation on decision type

File: db/migrate/20200513105319_create_srg_histories.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+      t.timestamps
+    end
+
+    add_column :eu_decisions, :srg_history_id, :integer
+    add_foreign_key :eu_decisions, :srg_histories, name: 'eu_decisions_srg_history_id_fk'
+
+    execute(<<-SQL
+      INSERT INTO srg_histories(name, tooltip, created_at, updated_at)
+      VALUES ('In consultation', NULL, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP), ('Discussed at SRG (no decision taken)', NULL, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP)
```
Comment (lucacug) [inline db/migrate/20200513105319_create_srg_histories.rb:15]: I would move the "no decision taken" bit of the DIscussed at SRG value to the tooltip?

File: app/controllers/admin/eu_decision_types_controller.rb [hunk 2]
```diff
@@ -8,12 +8,6 @@ def index
     end
   end
 
-  def create
```
[the remark this answers is not in the document]
Comment (Levia) [reply app/controllers/admin/eu_decision_types_controller.rb:11]: same!

File: db/migrate/20200513105319_create_srg_histories.rb (hunk 1 above)
Comment (Levia) [reply db/migrate/20200513105319_create_srg_histories.rb]: Yeah, it might be better actually, thanks!
That's done now

**Comment by lucacug** [comment]
Happy with this. Merging.

**Pull request merged**